### ISSUE TITLE
Cycling windows in Monocle Layout with minimization turned on

### ISSUE THREAD

**Issue by phddoom** [esjeon/krohnkite#78]
**Symptom**

What is the expected workflow for cycling windows with the `Minimize unfocused windows` option?

**How to Reproduce**
1. Set `Minimize unfocused windows` option for Monocle Layout
2. Press key to cycle/move focus

**Expected behavior**

I expected with this option selected that using the keybindings to cycle or move focus would cycle through all the windows on the current desktop.

**Environment**
 - Distro: Archlinux
 - KWin version: 5.18.4.1
 - Krohnkite version: git commit 18e8b62
 - List of KWin scripts in use: Force Blur, Move Window to Center, Synchronize Skip Switcher with Taskbar

**Notes**

**Comment by esjeon** [comment]
Yup, you're right, and this is why it is marked WIP. I'll definitely fix this.

**Comment by zyf0330** [comment]
When using Monocle Layout with many windows openning, I minimize some of them to prevent cycle/move focus into these windows. And in another way I don't know what situation `Minimize unfocused windows` option works for.

**Comment by matejdro** [comment]
I see two use cases for this features:

* Some programs might release resources / stop rendering UI when being minimized and thus wasting less resources when in background
* Task Manager widget has an option to filter out minimized windows. That way you would only see background windows if minimization was enabled.

**Comment by zyf0330** [comment]
@matejdro You are right. But hope that disable this option won't break the behavior that not switch to minimized window.
